docs/plugins.md

```diff
@@ -48,6 +48,18 @@ application without IPC and without forking Kubo.
 Note: We eventually plan to make Kubo usable as a library. However, this
 plugin type is likely the best interim solution.
 
+### fx
+
```

Comment:
LGTM, although I'd probably add some tag like `(never stable)` or `(experimental)` here as well given that we may still evolve what goes in here.

It's likely also worth calling out, whether here or in the plugin interface code the comments in https://github.com/ipfs/kubo/blob/837f6ee92b6ee00e7c7e364084fa93959073c8c7/core/builder.go#L28-L42 since IIUC where they are right now is unlikely to be seen by people building new plugins.